shared/base/view.js

```diff
@@ -195,7 +195,7 @@ module.exports = BaseView = Backbone.View.extend({
    * Get the HTML for the view, including the wrapper element.
    */
   getHtml: function() {
-    var html = this.getInnerHtml(),
+    var html = this.options.lazy ? '' : this.getInnerHtml(),
```

Comment:
So, I took [your comment](https://github.com/rendrjs/rendr/pull/488/files#r35947443) as suggesting that rendr-handlebars' `getClientPlaceholder` function should be invoked on the server to get the outer html placeholder div that is eventually replaced client-side.  But, that doesn't work for me,(I get `TypeError: Converting circular structure to JSON` from [this line](https://github.com/rendrjs/rendr-handlebars/blob/master/shared/helpers/view.js#L70)) and it also appears that `getClientPlaceholder` is a function that is called client-side, not server-side https://github.com/rendrjs/rendr-handlebars/blob/master/shared/helpers/view.js#L31-L36